WordPress/Classes/ViewRelated/Site Creation/Site Name/SiteNameView.swift

```diff
@@ -83,10 +84,14 @@ class SiteNameView: UIView {
         button.translatesAutoresizingMaskIntoConstraints = false
         button.titleLabel?.font = WPStyleGuide.fontForTextStyle(.body, fontWeight: .semibold)
         button.setTitle(TextContent.continueButtonTitle, for: .normal)
-        // TODO: SITENAME - This button is still missing a target, development is WIP.
+        button.addTarget(self, action: #selector(navigateToNextScreen), for: .touchUpInside)
         return button
     }()
 
+    @objc private func navigateToNextScreen() {
+        onContinue(searchBar.text)
```

Comment:
Should we take this opportunity (or somewhere up the chain - factory closure, `didStep`) to trim the string?

Reply:
Sure! Just pushed a change for it.